ARExperiment/ARExperiment/ARExampleViewController.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+class ARExampleViewController: UIViewController {
+
+    func setbackgroundColor() {
```

Comment:
you could have these as extensions on UIView controller, instead of requiring a subclass. That way, we avoid large class hierarchies.

also note you need a capital B in background on this method

Reply:
If I had extensions then I still need to set it on every VC and I wanted to avoid having to do that since is how you forget to do it sometimes, especially since more than one people mantain this. 

That was my goal but I do not like subclassing either 🤷‍♀️

Reply:
[you are still setting it though](https://github.com/novoda/ios-demos/pull/31/files#diff-784d1c4ad2b72af192472216ab006ecbR30) 😉

Reply:
well, strictly not setting it, but you still have to invoke it. So that should be in an extension, because then it is (partial) [composition over inheritance](https://medium.com/humans-create-software/composition-over-inheritance-cb6f88070205)

Reply:
I did not remember that we were still setting it 🤦‍♂️